core/ops_metrics.rs

```diff
@@ -11,10 +11,12 @@ pub struct OpMetrics {
   pub ops_dispatched: u64,
   pub ops_dispatched_sync: u64,
   pub ops_dispatched_async: u64,
+  // TODO(bartlomieju): this field is never updated
   pub ops_dispatched_async_unref: u64,
   pub ops_completed: u64,
   pub ops_completed_sync: u64,
   pub ops_completed_async: u64,
+  // TODO(bartlomieju): this field is never updated
```

Comment:
What is the reason for not removing them? Are they considered part of a public or stable(ish) API?

Reply:
Yes, this are part of stable `Deno.Metrics` interface